### PR TITLE
feat(RichtTextContent): added support for links and line breaks

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Added support for links and line breaks on `RichTextContent` component

## Justify why they are needed

Requested for used on AccordionBlock

Screenshot 2023-01-30 at 15 57 41

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1488** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1488" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1488?utm_source=stack-comment).

File: apps/store/src/components/RichTextContent/RichTextContent.tsx [hunk 1]
```diff
@@ -24,4 +24,8 @@ export const RichTextContent = styled.div({
   '& ol': {
     listStyle: 'decimal',
   },
+  '& a': {
```
Comment (alebedev) [inline apps/store/src/components/RichTextContent/RichTextContent.tsx:27]: Do we also need some custom styles for hover, active, visited states?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/RichTextContent/RichTextContent.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/components/RichTextContent/RichTextContent.tsx:27]: Checked with Zak. I've properly updated it to match the expected design.

**Pull request merged**